app/models/revenue.rb

```diff
@@ -0,0 +1,8 @@
+class Revenue
```

Comment:
In this iteration revenue is just here to hold data, still leaving the calculations on the Merchant model